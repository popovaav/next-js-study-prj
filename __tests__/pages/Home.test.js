import { render } from '../settings/test-utils';
import '@testing-library/jest-dom/extend-expect';
import Home from '../../pages/index';

test('renders a message', () => {
  const { container, getByText } = render(<Home />);
  const linkElement = getByText(
    /welcomeMessage/
  );
  expect(linkElement).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
