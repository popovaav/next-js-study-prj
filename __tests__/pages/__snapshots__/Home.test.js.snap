// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="main"
    >
      <div
        class="page"
      >
        <h1
          class="title"
        >
          welcomeMessage
           
          <a
            href="https://nextjs.org"
          >
            nextJs
            !
          </a>
        </h1>
        <p
          class="description"
        >
          getStarted
           
          <code
            class="code"
          >
            pages/index.js
          </code>
        </p>
        <div
          class="grid"
        >
          <a
            class="card"
            href="https://nextjs.org/docs"
          >
            <h3>
              documentation
               →
            </h3>
            <p>
              findInformation
            </p>
          </a>
          <a
            class="card"
            href="https://nextjs.org/learn"
          >
            <h3>
              learn
               →
            </h3>
            <p>
              learnAbout
            </p>
          </a>
          <a
            class="card"
            href="https://github.com/vercel/next.js/tree/master/examples"
          >
            <h3>
              examples
               →
            </h3>
            <p>
              discoverAndDeploy
            </p>
          </a>
        </div>
      </div>
    </main>
    <footer
      class="footer"
    >
      <a
        href="https://onix-systems.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        poweredBy
         
        onix
      </a>
    </footer>
  </div>
</div>
`;
